include/vkgcDefs.h

```diff
@@ -393,6 +394,12 @@ enum class ThreadGroupSwizzleMode : unsigned {
   Count
 };
 
+/// Represents mapping layout of the resources used in shaders
+enum class ResourceLayoutScheme : unsigned {
+  Compact = 0, ///< Compact scheme make full use of all the user data registers.
+  Indirect     ///< Fixed layout, push constant will be the sub node of DescriptorTableVaPtr
```

Comment:
When VK_PIPELINE_LAYOUT_CREATE_INDEPENDENT_SETS_BIT_EXT is used, we will using indirect mode.

Reply:
Maybe a full document is not needed, but at least move the comment you have the EntryPointMutate.cpp here.  This will make it easier to find.  If someone wants to find out what the indirect scheme is, they look at the definition of the enum.